test/traditional/delaytime_test.jl

```diff
@@ -13,14 +13,15 @@ diffeq = (atol = 1e-9, rtol = 1e-9)
     @test exponential_decay_fit(x, y, :small) ≈ 5
 
     henon_rule(x, p, n) = SVector(1.0 - p[1]*x[1]^2 + x[2], p[2]*x[1])
-    ds = DeterministicIteratedMap(henon_rule, zeros(2), [1.4, 0.3])
+    ds = DiscreteDynamicalSystem(henon_rule, zeros(2), [1.4, 0.3])
 
-    data = trajectory(ds, 1000)[1]
+    data = trajectory(ds, 1000)#[1]
```

Comment:
This is an odd change... You are going from DynamicalSystems.jl v3 to v2? Why? No regressions should happen in this PR. Please update your local code to DYnamicalSystems.jl v3 and adjust accordingly.